src/meta_ext.rs

```diff
@@ -229,6 +232,17 @@ impl<'a> Pallet<'a> for PalletMeta {
         let name = self.name.decoded();
         name
     }
+
+    fn get_calls(&self) -> Option<&PalletCallMetadata<PortableForm>> {
+        self.calls.iter().find(|_call| {
+            true
```

Comment:
? what's the purpose of the find with the `true`?

Reply:
That was due to a lack of `.first()` on that iterable. The `true` acts as a find that first. I rewrote this part to make it look cleaner.